packages/plugin-amplitude-node/lib/index.ts

```diff
@@ -51,8 +49,8 @@ export class AmplitudePlugin extends RequestLoggerPlugin {
     this.amplitude = new Amplitude(this.apiKey, this.options);
   }
 
-  async identify(userId: string, properties?: Properties, options?: IdentifyOptions) {
-    const { callback } = this.getPluginCallOptions<AmplitudeIdentifyOptions>(options);
+  async identify(userId: string, properties?: Properties, options?: AmplitudeIdentifyOptions) {
```

Comment:
The typecast here is awesome. I was hoping we could do that but wasn't sure if we could.